src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -231,6 +230,11 @@ protected override void OnApplyTemplate()
 				_deleteButton = new WeakReference<Button>(button);
 			}
 
+			if (_contentElement is { })
+			{
+				_contentElement.SetProtectedCursor(Microsoft/* UWP don't rename */.UI.Input.InputSystemCursor.Create(Microsoft/* UWP don't rename */.UI.Input.InputSystemCursorShape.IBeam));
+			}
```

Comment:
This doesn't seem relevant to reveal button, and unclear what this change is trying to fix.

Reply:
It limits the IBeam cursor change to the "textbox body" and excludes the reveal/delete buttons.